plugins/modules/content_import_library.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2024, Partha Aji
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: content_import_library
+version_added: 4.0.0
+short_description: Manage library content imports
+description:
+    - Import library content to foreman.
+author:
+    - "Partha Aji (@parthaa)"
+options:
+  path:
+    description:
+      - Directory containing the exported library.
+    required: true
+    type: str
+  metadata:
+    description:
+      - Contents of the metadata.json file. This is not required if the metadata_file location is provided.
+  metadata_file:
+    description:
+      - Location of the metadata.json file. Not required if the metadata has been already provided via the other parameter
```

Comment:
I think all three options could live in a documentation fragment, so we don't have to repeat ourself.

Reply:
do you have examples of document fragments?

Reply:
https://github.com/theforeman/foreman-ansible-modules/blob/d195da2d1e6c4c766739e5fdf4156bc585e91807/plugins/doc_fragments/foreman.py#L390-L417
used e.g. here:
https://github.com/theforeman/foreman-ansible-modules/blob/d195da2d1e6c4c766739e5fdf4156bc585e91807/plugins/modules/content_export_library.py#L45